css-structure/css-structure_selectors-and-cascading/css/styles.css

```diff
@@ -49,93 +49,79 @@ nav li {
 /* -------------------------------------------------------- */
 
 /* 1. These styles should apply to the headline */
-FIX_ME {
-  outline: 2px dashed red;
+header {
   font-size: 2rem;
-  margin: 0 2rem 0 0;
+  margin: 0;
   color: #2673bb;
 }
 
 /* 2. These styles should apply to all links */
-FIX_ME {
-  outline: 2px dashed red;
+a {
   color: #ff00e6;
 }
 
 /* 3. These styles should apply to all external links (you can use the attribute selector) */
-FIX_ME {
-  outline: 2px dashed red;
+[target] {
   color: #ff8000;
 }
 
 /* 4. These styles should apply to all links in the main navigation */
-FIX_ME {
-  outline: 2px dashed red;
+nav a {
   color: #ffffff;
   text-decoration: none;
 }
 
 /* 5. These styles should apply to the main navigation link corresponding to the current page */
-FIX_ME {
-  outline: 2px dashed red;
+nav a.current {
   color: #22e781;
 }
 
 /* 6. These styles should apply to all links in the footer */
-FIX_ME {
-  outline: 2px dashed red;
+footer a {
   color: inherit;
 }
 
 /* 7. These styles should apply to all paragraphs and all lists (please use a class selector) */
-FIX_ME {
-  outline: 2px dashed red;
+.list .text {
   color: #2c3f3f;
 }
 
 /* 8. These styles should apply to all "notes" */
-FIX_ME {
-  outline: 2px dashed red;
+.text.text-note {
   font-size: 0.8em;
   color: #808080;
 }
 
 /* 9. These styles should apply to the welcome text */
-FIX_ME {
-  outline: 2px dashed red;
+.text#welcome-text {
   font-size: 1.2rem;
   color: #000000;
 }
 
 /* 10. These styles should apply to the paragraph in the footer */
-FIX_ME {
-  outline: 2px dashed red;
+footer .text {
```

Comment:
Here might `footer p.text` be right?

Reply:
Not necessarily, if a class is set then the tag together with the class does not need to be accessed, unless there are other tags with the same class in the block, but this is bad practice.